index.md

```diff
@@ -0,0 +1,39 @@
+# Node Disk Manager Helm Repository
+
+<img width="300" align="right" alt="OpenEBS Logo" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/stacked/color/openebs-stacked-color.png" xmlns="http://www.w3.org/1999/html">
+
+[Helm3](https://helm.sh) must be installed to use the charts.
+Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+
+Once Helm is set up properly, add the repo as follows:
+
+```bash
+$ helm repo add ndm https://openebs.github.io/node-disk-manager
+```
+
+You can then run `helm search repo ndm` to see the charts.
+
+#### Update OpenEBS Repo
+
+Once OpenEBS repository has been successfully fetched into the local system, it has to be updated to get the latest version. The OpenEBS repo can be updated using the following command.
+
+```bash
+helm repo update
+```
+
+#### Install using Helm 3
```

Comment:
Should we mention as `Helm3` here. Because We have already said that Helm3 is required to install the charts.

Reply:
I am resolving this comment -- assuming no action item needs to be taken here. 
Please unresolve and comment if there are any.